src/styles/_catalog.scss

```diff
@@ -16,6 +16,12 @@
     margin-bottom: 48px;
   }
 
+  &__card:hover {
+    transform: scale(120%);
+    color: #00acdc;
+    transition: 300ms;
```

Comment:
`transition: 300ms` better place in &__card selector

Reply:
catalog__card--hover do ypu mean like this one?